src/GraphQL.MemoryCache/MemoryDocumentCacheOptions.cs

```diff
@@ -0,0 +1,29 @@
+using System;
+using Microsoft.Extensions.Caching.Memory;
+using Microsoft.Extensions.Options;
+
+namespace GraphQL.Caching
+{
+    /// <summary>
+    /// Provides configuration options for <see cref="MemoryDocumentCache"/>.
+    /// </summary>
+    public class MemoryDocumentCacheOptions : MemoryCacheOptions, IOptions<MemoryDocumentCacheOptions>
+    {
+        public MemoryDocumentCacheOptions()
+        {
+            SizeLimit = 100000;
```

Comment:
`SizeLimit` and `MaxTotalQueryLength` both means the same. When I showed my example at the very beginning of the review, I had it in mind even then, just then I didn't notice that inheritance was lost.